### PR TITLE
docs(template): PR template attribution to comment

### PR DESCRIPTION
**Pull request by d33bs**
# Description

This PR makes a quick adjustment to the PR template to avoid a step which is performed each time a PR is opened on GitHub to remove the template credit text. Moving to an HTML comment for this text retains the credit in the PR text without rendering it when reading the PR text, removing a step for maintainers to remove the content each time a PR is opened (and persisting the credit).

Thank you for any feedback you may have!

Related to and ported from: https://github.com/cytomining/CytoTable/pull/139#pullrequestreview-1809744933

## What is the nature of your change?

- [ ] Bug fix (fixes an issue).
- [x] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

**Review by gwaybio — APPROVED** [review]

**Review by kenibrewer — APPROVED** [review]

**Comment by d33bs** [comment]
Thank you @gwaybio and @kenibrewer ! May I merge this in?

**Comment by gwaybio** [comment]
Yep, you are a maintainer and this PR has at least one approval by a maintainer 👍

**Pull request merged**
